gherkin/perl/dist.ini

```diff
@@ -27,8 +27,17 @@ repository.type   = git
 [PkgVersion]
 [Prereqs::FromCPANfile]
 [Git::GatherDir]
-exclude_filename=VERSION
+exclude_match=^helper-scripts/
+exclude_filename=gherkin.berp
+exclude_filename=gherkin-perl.razor
+exclude_filename=gherkin-languages.json
```

Comment:
Are we sure we want to excelude `gherkin-languages.json`? Perl implementation do not use it at runtime?

Reply:
That's correct: the file only needs to exist at build time: https://github.com/cucumber/common/blob/main/gherkin/perl/helper-scripts/build_languages.pl generates the lib/Gherkin/Generated/Languages.pm file which is a compressed Perl representation of the JSON file's content. The idea is that it's much easier and more standard to deploy Perl files than "other" files.